packages/preview/usaf-memo/0.0.1/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "usaf-memo"
+version = "0.0.1"
+compiler = "0.13.0"
+entrypoint = "lib.typ"
+repository = "https://github.com/SnpM/typst-usaf-memo"
+authors = ["Nibs <https://github.com/snpm>"]
+license = "MIT"
+description = "Typeset memos that are fully compliant with AFH 33-337 'The Tongue and Quill'. Under development."
+keywords = ["USAF", "Air Force", "memorandum", "memo", "military", "AFH 33-337", "official", "correspondence"]
+categories = ["office", "report"]
+disciplines = ["politics", "law", "business", "engineering", "education"]
+
+[template]
+path = "template"
+entrypoint = "starkindustries.typ"
```

Comment:
The default content is fine, but I think it would be better to have a more generic file name, so that people can just delete the default contents and replace it with their own, without having to rename the file to something more meaningful.